editor/js/Sidebar.Scene.js

```diff
@@ -349,7 +350,7 @@ function SidebarScene( editor ) {
 
 				}
 
-                // Hide utility scene objects and labels from materials editor
+				// Hide utility scene objects and labels from materials editor
```

Comment:
Let's remove whitespace

Reply:
It's automatically formatted via eslint, so I can't rollback to previous version (except disabling linting for this specific comment, which is, obviously, not a good idea)